### DISCUSSION TITLE
生成 AI 被超車？Google 大神偶包太沉重

### DISCUSSION THREAD

**Discussion by Lin-NZ** [1101-GS3073-C/Python-Community-GS3073-C#725] [General]
Content:
https://www.inside.com.tw/article/30392-google-microsoft-ai-chatgpt

Reflections / Thoughts:
過去Google靠著新技術擴大網路效應創造收入，但OpenAI利用了這些技術，創造超越了Google本身產品的ChatGPT
當主要的商業模式:廣告搜尋開始受到威脅時，Google是否還能保持無私的態度，將新技術無償分享給大眾及其他巨頭呢?還是開啟新的知識類營利模式?值得觀察。

**Comment by alvin1226** [comment]
我認為這是件好事，彼此有競爭才能讓對方進步，但這必須建立在不傷害彼此上

**Comment by yjpxi** [comment]
未來的趨勢將取決於 Google 對市場變化的回應方式，如果 Google 能夠成功轉型，找到新的盈利模式並保持技術領先地位，它仍有可能繼續在開放與盈利之間取得平衡。
